core/src/App.html

```diff
@@ -1,5 +1,8 @@
 <svelte:window on:resize="{onResize}" />
-<div id="app" class="{hideNav? 'no-nav' : ''} {hideSideNav? 'no-side-nav':''}">
+<div
+  id="app"
```

Comment:
In general we try to avoid to use `id`'s on html elements, because we offer a framework and don't want to "block" ids  for customers project. If the customer use the same id he will get unforeseen side effects. Especially`app` is most likely an id which will be used by someone else.

Reply:
You probably noticed that I've maintained the status quo. 
Apart from me, ```app``` was already used by some other CSS. :)
I was assuming that this id is used for some booting but I can only see one reference to ```app``` and it is in the vue.js example - would it use the ```id="app"``` in ```App.html``` to boot?
Shall I refactor or would this be another ticket?